src/postgres-util/src/lib.rs

```diff
@@ -29,8 +28,39 @@ use crate::desc::{PostgresColumnDesc, PostgresTableDesc};
 
 pub mod desc;
 
+/// An error that happened during purification.
+/// This should be comprehensible (and actionable)
+/// by a user.
```

Comment:
I think this is copypasta from an earlier version of this PR?

Reply:
ugh, sorry about that!